CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# CONTRIBUTING
+
+We're accepiting help from anyone! This repo is collaborative, so if you want to add a solution in a new language (check Wikipedia [Programming Language List](https://en.wikipedia.org/wiki/List_of_programming_languages)) or just improve some code that you think that can be improved, follow this steps:
+
+
+## 1. Fork it :fork_and_knife:
+
+You can get your own fork/copy of [Stalin Sort](https://github.com/gustavo-depaula/stalin-sort) by using the <a  href="https://github.com/Hacktoberfest-2018/Hello-world/new/master?readme=1#fork-destination-box"><kbd><b>Fork</b></kbd></a> button or clicking [this](https://github.com/gustavo-depaula/stalin-sort/new/master?readme=1#fork-destination-box).
+
+  [![Fork Button](https://help.github.com/assets/images/help/repository/fork_button.jpg)](https://github.com/gustavo-depaula/stalin-sort/new/master?readme=1#fork-destination-box)
+
+## 2. Start coding! :rabbit2:
+
+Once you have forked the repo, add your program in the language folder in main branch, if there is no language folder, make one, then add into it.
+
+For a PR to be approved and merged please note:
+
+- We allow more than one solution for one language, so if you don't want to modify someones code, you can add a new file with your own solution. Have in mind that your solution must be completely unique, if it isn't consider modify the existing code.
+- We allow maximum of 5000 additions (your PR will get close if its exceeds, but you can ask maintainer for reviewing this)
+- We don't allow deletions of others files or code.
+- We don't allow files to be directly added in root directory of project, add them in there language folder
+- Follow the code specifications!
+
+### Code specifications
+
+For the purpuses of this repo, we defined some rules that your code must to follow, this rules are listed bellow:
+1. Your file **must** be named **stalin-sort**. All the other files of the same language must follow a numeric order, so the second implementation **must** be named as **stalin-sort-2**.
```

Comment:
this is not helpful. if we have more than one version of the same language, is better to have it named like "stalin-sort-compile-time" so we can know what it does from the name